runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -455,4 +466,10 @@ private LootTrackerItem buildLootTrackerItem(int itemId, int quantity)
 
 		return trackerRecords;
 	}
+
+	LootTrackerValueType getValueType()
+	{
+		return config.valueType();
+	}
+
```

Comment:
remove this empty line

Reply:
de0366819507da1b1d7aa15cc39f99176f5f9c62